services/web/server/src/simcore_service_webserver/cli.py

```diff
@@ -40,6 +36,9 @@
 def _setup_app_from_settings(
     settings: ApplicationSettings,
 ) -> tuple[web.Application, dict]:
+    from .application import create_application
+    from .application_settings_utils import convert_to_app_config
+    from .log import setup_logging
```

Comment:
Why are these imported here? I find it a bit out of place?

Reply:
yes. i was trying to speed up startup (see EXTRA above)
It is a small i mprovement

Reply:
could you please add a comment near the imports here and down below hinting to these?
Sorry the extra hint was not that helpful for me:(